superdesk/editor_utils.py

```diff
@@ -401,29 +465,170 @@ def __init__(self, item, field='body_html'):
         self.field = field
         data = item.get('fields_meta', {}).get(field, {})
```

Comment:
I don't remember why exactly it was done this way, but I think that it should be a `setdefault` here instead of `get`, otherwise `fields_meta` won't be generated if it was not already present in the item (which can happen even for editor 3 on just ingested or unmodified articles)